wolfssl/wolfcrypt/asn_public.h

```diff
@@ -759,6 +759,12 @@ typedef struct _wc_CertPIV {
 WOLFSSL_API int wc_ParseCertPIV(wc_CertPIV* cert, const byte* buf, word32 totalSz);
 #endif /* WOLFSSL_CERT_PIV */
 
+/* Forward declaration needed, as DecodedCert is defined in asn.h.*/
+struct DecodedCert;
```

Comment:
Users will still need to include our `wolfssl/wolfcrypt/asn.h` to be able to deference `DecodedCert`. But at least there will be visible API's now!